src/screens/planItemActivity/TimeSlider.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { SFC, useState } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+
+import { Slider } from 'components';
+import { i18n } from 'locale';
+import { palette } from 'styles';
+
+interface Props {
+  min: number;
+  max: number;
+  closeModal?: () => void;
+  onConfirm?: (time: number) => void;
+}
+
+export const TimeSlider: SFC<Props> = ({ min, max, closeModal, onConfirm }) => {
```

Comment:
isn't `SFC` deprecated in favor of `FC`? With react hooks it's no longer "stateless" ;)